ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/PartyLedgerEntry.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.stores.ledger
+
+import java.time.Instant
+
+import com.daml.ledger.participant.state.v1.{ParticipantId, SubmissionId}
+import com.digitalasset.ledger.api.domain.PartyDetails
+
+sealed abstract class PartyLedgerEntry() extends Product with Serializable {
+  val submissionIdOpt: Option[SubmissionId]
+  val recordTime: Instant
```

Comment:
```suggestion
  def recordTime: Instant
```
Unless there's a good reason to do otherwise, of course. :slightly_smiling_face:

Reply:
Not necessary to define as def here because it is not used in the base class' constructor.